src/window.cc

```diff
@@ -656,6 +656,64 @@ void Window::set_menu_actions() {
     SelectionDialog::get()->show();
   });
   
+  menu.add_action("source_find_file", [this]() {
+    auto view = Notebook::get().get_current_view();
+    auto project_path = boost::filesystem::canonical(Directories::get().path);
```

Comment:
See https://github.com/cppit/jucipp/blob/master/src/window.cc#L599 on how to find the path in which to search.